utils/plots.py

```diff
@@ -0,0 +1,152 @@
+import pandas as pd
+import numpy as np
+import seaborn as sns
+import matplotlib.pyplot as plt
+from sklearn.metrics import r2_score
+from sklearn.feature_selection import r_regression
+from scipy.stats import spearmanr
+from scipy.stats import pearsonr
+
+
+def plot_multiple_cdfs_with_medians(num_of_top_preds, dataframes_for_top_preds, labels_for_top_preds, lfc_values_list, labels, title_sufix):
+    """
+    Plots the CDFs for multiple sets of log2 fold change (LFC) values with median lines from multiple dataframes.
+    
+    Parameters:
+    - num_of_top_preds: Number of top predictions for each miRNA
+    - dataframes_for_top_preds: List of DataFrames, each containing 'miRNA', predictions (denoted by label), and fold changes (denoted by 'fold_change'). Only top N predictions for miRNA will be plotted
+    - labels_for_top_preds: List of prediction labels
+    - lfc_values_list: List of arrays of LFC values, all values will be plotted
```

Comment:
Why are lfc_values_list and labels as separate input parameters and not part of dataframes_for_top_preds? Maybe it makes sense with connection to other parts, just asking.

Reply:
The difference is that lfc_values_list FC values are plotted as a whole, whereas for dataframes_for_top_preds we pick the top 16 predictions and plot their FC. 
The lfc_values_list serves for real FC values. 
The dataframes_for_top_preds is for predictions.
It is a generic function where you can plot only one of the groups or their combination. 
I renamed the variables and updated the function description to make it more clear.